cmd/root.go

```diff
@@ -91,16 +108,20 @@ func Execute() {
 }
 
 func init() {
-	RootCmd.Flags().StringP("owner", "o", "", "target repository owner")
```

Comment:
Can you please update the examples in the `README` such that they no longer use the `--owner` flag?